tests/SystemFunctionsTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace BeyondCode\SelfDiagnosis\Tests;
+
+use BeyondCode\SelfDiagnosis\SystemFunctions;
+use PHPUnit\Framework\TestCase;
+
+class SystemFunctionsTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function it_find_an_available_function()
+    {
+        $systemFunctions = new SystemFunctions();
+        $this->assertFalse($systemFunctions->isFunctionAvailable('print'));
```

Comment:
The assert does not match the function description, I think.

Reply:
Good point, the test was incorrect, I fixed it in f3e3815dfa5e7660a4bd3c9bc4fae1cd1213e153